activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -346,6 +346,14 @@ def test_include_multiple_columns_index
             assert_equal ["number", "data"], index.include
           end
         end
+
+        def test_include_keyword_column_name
+          with_example_table("id integer, timestamp integer") do
+            @connection.add_index "ex", :id, name: "include", include: [:timestamp]
+            index = @connection.indexes("ex").find { |idx| idx.name == "include" }
+            assert_equal ["timestamp"], index.include
+          end
+        end
       end
 
```

Comment:
Could be a good idea to add a regression test for quotes in column names:

```suggestion

      def test_include_escaped_quotes_column_name
        with_example_table(%{id integer, "I""like""quotes" integer}) do
          @connection.add_index "ex", :id, name: "include", include: [:"I\"like\"quotes"]
          index = @connection.indexes("ex").find { |idx| idx.name == "include" }
          assert_equal ["I\"like\"quotes"], index.include
        end
      end
```